zcash_test_vectors/orchard/asset_id.py

```diff
@@ -70,12 +83,12 @@ def randbytes(l):
 
         key_bytes = bytes(fvk.ivk())
         description_bytes = get_random_unicode_bytes(512)
-        asset = asset_id(key_bytes, description_bytes)
+        asset = asset_base(asset_digest(encode_asset_id(key_bytes, description_bytes)))
```

Comment:
The naming here and below is inconsistent. should be
```python
asset_digest = asset_digest(encode_asset_id(key_bytes, description_bytes))
asset_base = asset_base(asset_digest)
```

Reply:
Changed it to `asset_base = asset_base(asset_digest(encode_asset_id(key_bytes, description_bytes)))`